docs/source/considerations.mdx

```diff
@@ -0,0 +1,42 @@
+
+If the dataset you're using doesn't have a predefined train-test split, it is up to you to define which part of the dataset you want to use for training your model and which you want to use for evaluation (since training and evaluating on the same split can misrepresent your results!). Depending on the size of the dataset, you can keep anywhere from 10-30% for evaluation and the rest for testing. Check out [this thread](https://discuss.huggingface.co/t/how-to-split-main-dataset-into-train-dev-test-as-datasetdict/1090) for a more in-depth discussion of dataset splitting!
+
+## The impact of class imbalance
+
+In the case of supervised learning tasks (i.e. tasks for which labels exist), an *imbalance* in the classes can impact evaluation results -- this means that if some classes have much more examples than others, these can impact evaluation results and misrepresent performance.
+
+Often, using more than one metric can help get a better idea of your model’s performance from different points of view. For instance, metrics like **[accuracy](https://huggingface.co/metrics/accuracy)** and **[precision](https://huggingface.co/metrics/precision)** can be used together, and the **[f1 score](https://huggingface.co/metrics/f1)** is actually the harmonic mean of the two.
+
+In cases where a dataset is balanced, using accuracy can be reflect the overall model performance:
+
+![Balanced Labels](media/balanced-classes.png)
+
+In cases where there is an imbalance, using F1 score can be a better representation of performance:
+
+![Imbalanced Labels](media/imbalanced-classes.png)
+
+## Offline vs. online model evaluation
+
+Offline model evaluation is any model evaluation that you do using offline data before you deploy a model or use any insights generated from a model.
+
+online model evaluation refers to the process of evaluating how your model is performing after it has actually been deployed and is being used in production.
+
+## Trade-offs in model evaluation
+
+### Interpretability
+
+Not all regression metrics are easy to interpret on their own. For example, it is hard to say what a good value for RMSE is because that depends on the scale of your outcome variable. If your outcome variable ranges from 0 to 1 then a model with a MAE of 0.8 would be considered bad, whereas if your outcome variable ranges from -500 to 500 then a MAE of 0.8 would be considered great. If you want your error metrics to be interpretable out of the box, then you should use error metrics that are scaled to account for the range of your outcome variable such as mean absolute percent error (MAPE).
```

Comment:
Maybe we could start with a general statement and then go into regression as an example? Reading it sounds a bit like interpretability is only an issue with regression and is solved with MAPE - what's the big deal :P